include/grpc/grpc_security.h

```diff
@@ -812,49 +812,77 @@ GRPCAPI int grpc_tls_key_materials_config_set_key_materials(
 
 /** Set grpc_tls_key_materials_config instance with a provided version number,
     which is used to keep track of the version of key materials.
-    It returns 1 on success and 0 on failure. It is used for
-    experimental purpose for now and subject to change.
+    It returns 1 on success and 0 on failure.
+    Note the API is experimental now and subject to change.
  */
 GRPCAPI int grpc_tls_key_materials_config_set_version(
     grpc_tls_key_materials_config* config, int version);
 
 /** Get the version number of a grpc_tls_key_materials_config instance.
     It returns the version number on success and -1 on failure.
-    It is used for experimental purpose for now and subject to change.
+    Note the API is experimental now and subject to change.
  */
 GRPCAPI int grpc_tls_key_materials_config_get_version(
     grpc_tls_key_materials_config* config);
 
 /** --- TLS credential reload config. ---
-    It is used for experimental purpose for now and subject to change.*/
+    Note the API is experimental now and subject to change.*/
 
 typedef struct grpc_tls_credential_reload_arg grpc_tls_credential_reload_arg;
 
 /** A callback function provided by gRPC to handle the result of credential
     reload. It is used when schedule API is implemented asynchronously and
-    serves to bring the control back to grpc C core. It is used for
-    experimental purpose for now and subject to change. */
+    serves to bring the control back to grpc C core.
+    Note the API is experimental now and subject to change. */
 typedef void (*grpc_tls_on_credential_reload_done_cb)(
     grpc_tls_credential_reload_arg* arg);
 
 /** A struct containing all information necessary to schedule/cancel a
     credential reload request.
-    - cb and cb_user_data represent a gRPC-provided
-      callback and an argument passed to it.
+    Note that we currently only support synchronous credential reloading. All
+    the fields related to asynchronous credential reloading can be ignored for
+    now.
+    - cb and cb_user_data represent a gRPC-provided callback and an argument
+      passed to it.
+      They are expected to be used when credential reloading is done
```

Comment:
How about re-phrasing it as follows - "If users implement async credential reload, they must invoke `cb' by passing in `cb_user_data' at the end of reload operation. By doing that, gRPC C core stack will get informed about the completion of reload operation and can resume the handshake with the reloaded credential.".  `Expect' seems a little soft in my opinion.